src/Microsoft.Azure.WebJobs.Host.Storage/StorageServiceCollectionExtensions.cs

```diff
@@ -31,6 +31,15 @@ public static void AddAzureStorageCoreServices(this IServiceCollection services)
             // Adds necessary Azure services to create clients
             services.AddAzureClientsCore();
 
+            services.AddAzureStorageScaleServices(null);
+        }
+
+        public static void AddAzureStorageScaleServices(this IServiceCollection services, Action<JobHostInternalStorageOptions> configure)
+        {
+            if (configure != null)
+            {
+                services.Configure(configure);
```

Comment:
Why do we need to configure this SAS at all in the scaler scenarios? Scaler hosts won't even be using the infrastructure storage account, right?